package-lock.json

```diff
@@ -579,7 +583,7 @@
         "@octokit/request": "^9.1.1",
         "@octokit/request-error": "^6.1.1",
         "@octokit/types": "^13.4.1",
-        "lru-cache": "^10.0.0",
+        "lru-cache": "npm:@wolfy1339/lru-cache@^11.0.2-patch.1",
```

Comment:
?

Reply:
`@ctokit/auth-app` explicitly asks for this specific version:
https://github.com/octokit/auth-app.js/blob/7861dd553fe0b5cd0b4af25a38bbd86ff3df4069/package.json#L34

Reply:
See: https://github.com/octokit/auth-app.js/pull/651